jerry-core/jcontext/jcontext.h

```diff
@@ -193,6 +196,11 @@ struct jerry_instance_t
  */
 extern jerry_context_t jerry_global_context;
 
+/*
+* IF EXTERNAL_CONTEXT is not enable then INIT_CURRENT_INSTANCE is a empty macro to avoid warnings
```

Comment:
I recall that I've commented on this sentence earlier but cannot see the suggestion applied.